winml/test/image/imageTestHelper.cpp

```diff
@@ -53,14 +52,14 @@ namespace ImageTestHelper {
         com_ptr<ITensorNative> itn = tf.as<ITensorNative>();
         itn->GetBuffer(reinterpret_cast<BYTE**>(&pCPUTensor), &uCapacity);
         if (BitmapPixelFormat::Bgra8 == GetPixelFormat(modelPixelFormat)) {
-            for (UINT32 i = 0; i < size; i += 4) {
+            for (UINT32 i = 0; i < size - 2; i += 4) {
```

Comment:
since this is a non-trivial loop condition and increment, can you put a comment saying why?

Reply:
same with the other places